lib/static/modules/middlewares/metrika.js

```diff
@@ -9,14 +11,25 @@ export default metrikaClass => store => next => action => {
         case actionNames.INIT_STATIC_REPORT: {
             const startLoadTime = Date.now();
 
+            const {config: pluginConfig} = action.payload;
+            const ymConfig = get(pluginConfig, 'yandexMetrika', {});
+
+            if (!ymConfig.counterNumber) {
+                return next(action);
+            }
+
+            metrika = metrikaClass.create(ymConfig);
+
+            measurePerformance(({name, value}) => {
+                const intValue = Math.round(name === 'CLS' ? value * 1000 : value);
+                metrika.sendVisitParams({[name]: intValue});
+            });
+
             const result = next(action);
             const state = store.getState();
-            const config = get(state, 'config.yandexMetrika', {});
             const testsCount = get(state, 'tree.browsers.allIds.length', 0);
 
-            metrika = metrikaClass.create(config);
-
-            metrika.sendVisitParams({tti: Date.now() - startLoadTime, testsCount});
+            metrika.sendVisitParams({[action.type]: Date.now() - startLoadTime, testsCount});
```

Comment:
It is not a TTI metric, it is time of render main element of gui or static report.